lib/helpers.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
It would be nicer to place this file in `lib/helpers/expand-home-dir.js` instead of a generic *helper* file :)